OpenRA.Mods.Common/UpdateRules/Rules/DecoupleSelfReloading.cs

```diff
@@ -0,0 +1,83 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class DecoupleSelfReloading : UpdateRule
+	{
+		public override string Name { get { return "Replace 'SelfReloads' with 'ReloadAmmoPool'"; } }
+		public override string Description
+		{
+			get
+			{
+				return "'SelfReloads', 'SelfReloadDelay', 'ReloadCount', 'ResetOnFire'\n" +
+					"and 'RearmSound' were renamed and moved to a new 'ReloadAmmoPool' trait.";
```

Comment:
Neither Name above, nor this actually mentions they were moved from `AmmoPool`. Maybe just insert `from 'AmmoPool'` between the "move" and "to" here.

Reply:
Added.